src/nurbs.md

```diff
@@ -0,0 +1,55 @@
+tag-gettingstarted:
+
+# NURBS Miniapps
+
+These miniapps demonstrate the use of NURBS-based Isogeometric analysis[^1],[^2].
+
+[^1]: T.J.R. Hughes, J.A. Cottrell, Y. Bazilevs: "Isogeometric analysis: CAD, finite elements, NURBS, exact geometry and mesh refinement", Computer Methods in Applied Mechanics and Engineering, Elsevier, 2005, 194 (39-41), pp.4135-4195.
+
+[^2]: T.J.R. Hughes, J.A. Cottrell, Y. Bazilevs: "Isogeometric analysis: toward integration of CAD and FEA", Wiley&Sons 2009
+
+### NURBS Ex 1: Laplace problem
+
+This example code solves a simple Laplace problem
+\begin{align}
+-\Delta u = 1
+\end{align}
+ with
+homogeneous Dirichlet boundary conditions.
+
```

Comment:
Add links to the corresponding source code?

Reply:
Done.